src/lib.rs

```diff
@@ -1,13 +1,93 @@
-use zonebuilder;
-use pyo3::prelude::*;
+#[allow(unused)]
+use geo::{map_coords::MapCoordsInplace, Point};
+use pyo3::{prelude::*, types::PyDict, Python};
+use pythonize::pythonize;
+use zonebuilder::{self, Params};
 
 #[pyfunction]
 fn triangular_sequence(n: usize) -> Vec<f64> {
     zonebuilder::triangular_sequence(n)
 }
 
+#[pyclass]
+#[derive(Clone)]
+struct Pyparams {
+    num_segments: usize,
+    distances: Vec<f64>,
+    num_vertices_arc: usize,
+    precision: usize,
+    projected: bool,
+}
+impl Default for Pyparams {
+    fn default() -> Self {
+        Pyparams {
```

Comment:
How about calling `Params::default()` here and mapping back from that? Duplicating the values will be hard to maintain, especially across repositories